### DISCUSSION TITLE
Submodels + Import function?

### DISCUSSION THREAD

**Discussion by MrEdgeNL** [leozide/leocad#898] [Ideas]
Hello,

Would it be possible having an "import" function, next to the existing "export" function in the "SubModels..." menu?

I understand, when importing a LDR file, which also includes submodels, it starts to be fuzzy.
So in this case, importing everything till the first "NoFile" command would work?

Ofc. it's also possible to copy-paste a model/submodel from a second LeoCAD-instance, but this will NOT copy the STEP information.

Best regards,
Koos.

(Simular to this question: https://github.com/leozide/leocad/discussions/779)

**Comment by leozide** [comment]
Have you tried File -> Merge?

**Reply by MrEdgeNL** [reply]
Tnxs a lot, that works even better then I could think of!

Merging is creating submodels, where the "New Model" of the merged part, also acts like a submodel, which includes the other merged submodels. So it's possible to create nested subModels.. :)
